examples/webpack.config.local.js

```diff
@@ -73,13 +73,23 @@ const LOCAL_DEVELOPMENT_CONFIG = {
   },
 
   resolve: {
+    extensions: ['.ts', '.tsx', '.js', '.json'],
     // mainFields: ['esnext', 'browser', 'module', 'main'],
     // Imports the library from its src directory in this repo
     alias: Object.assign({}, ALIASES)
   },
 
   module: {
     rules: [
+      {
+        // Transpile ES6 to ES5 with babel
+        test: /\.js|ts$/,
```

Comment:
Would it be simpler to just add a new rule for `.ts`, rather than modify the rule for .js?